src/__tests__/index.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Following your response to https://github.com/la5ta/tally-web/pull/11#issuecomment-657282265, should we move this file to `/src/pages/index.test.tsx`?